src/js/components.js

```diff
@@ -11,13 +11,18 @@ class TimelineResultComp extends React.Component {
 }
 
 class TimelineButton extends React.Component {
-	render(){
-		return e('button', {className: this.props.className, key: 'timelineButton', onClick: this.props.onClickFunc}, this.props.buttonText);
+	render() {
+		return e('button', {className: this.props.className, key: 'timelineButton', onClick: this.props.onClickFunc, disabled: this.props.disabledFil}, this.props.buttonText);
```

Comment:
what is disableFil?

Reply:
disabling the filter button when there is nothing in the input box

Reply:
use disableFilter then versus abbreviating the word